--- a/cc/impls/linux_x86_64_linux_x86_64_cuda/wrappers/nvcc_wrapper
+++ b/cc/impls/linux_x86_64_linux_x86_64_cuda/wrappers/nvcc_wrapper
@@ -329,7 +329,7 @@ def main():
   parser.add_argument('-x', nargs=1)
   parser.add_argument('--cuda_log', action='store_true')
   args, leftover = parser.parse_known_args(sys.argv[1:])
-  args.cuda_log = True
+  args.cuda_log = False

   if args.x and args.x[0] == 'cuda':
     if args.cuda_log: log('-x cuda')
